x/fungible/keeper/evm.go

```diff
@@ -285,30 +284,28 @@ func (k Keeper) QueryZRC20Data(
 func (k Keeper) BalanceOfZRC4(
 	ctx sdk.Context,
 	contract, account common.Address,
-) *big.Int {
+) (*big.Int, error) {
 	abi, err := zrc20.ZRC20MetaData.GetAbi()
 	if err != nil {
-		return nil
+		return nil, sdkerrors.Wrapf(types.ErrABIUnpack, err.Error())
```

Comment:
lets switch to errors module from sdk errors ?

Reply:
8941403c